### ISSUE TITLE
Not clear solution to download/open in thrid part app a content (PDF, AVI, …)

### ISSUE THREAD

**Issue by kelson42** [openzim/nautilus#69]
Here, for example:
<img width="1024" alt="image" src="https://github.com/openzim/nautilus/assets/1029718/89f26aac-fda9-44bf-8f9b-aaf258976693">

on my FF, the only way to download one of this PDF would be to right or long click on one of the PDF logo.

**Comment by rgaudin** [comment]
The idea I think was that if there's PDF support in browser, browser will offers a download link aside viewing the PDF and that if there's no browser support, accessing triggers the download.

Obviously, one may want to _dump_ many files so having a download icon would help.
Additionally, mobile browsers don't expose download buttons directly (although it's available via the browser's menu).

@kelson42 would a download icon on the right work for you?

**Comment by kelson42** [comment]
> The idea I think was that if there's PDF support in browser, browser will offers a download link aside viewing the PDF and that if there's no browser support, accessing triggers the download.

Yes, but if not supported natively by the browser it should triggers „download or open via third party app“. i guess this is what you meant.

> Obviously, one may want to _dump_ many files so having a download icon would help. Additionally, mobile browsers don't expose download buttons directly (although it's available via the browser's menu).
> 
> @kelson42 would a download icon on the right work for you?

I suspect that this second part of the comment isnthe reason why I didn‘t answered straight… because I don‘t undesrtand it. What means „dump many files“ in this context?
